src/GoogleAuthenticator.php

```diff
@@ -82,7 +94,7 @@ public function authenticate($secret, $code)
 
     protected function getTimeSlice($offset = 0)
     {
-        return floor(time() / 30) + ($offset * 30);
+        return floor($this->time / 30) + $offset;
```

Comment:
The offset is multiplied by 30 because it's a time slice. Other offsets don't make sense with Google Authenticator.

Reply:
i'm not sure i understand well what's the problem here :)

Reply:
Sorry, I messed up. This is correct.